trailblazer/dto/summaries_response.py

```diff
@@ -4,10 +4,12 @@
 class Summary(BaseModel):
     order_id: int
     total: int
-    delivered: int
-    running: int
+
```

Comment:
Is this whitespace intended?

Reply:
For me it made sense to have this split, but if you want, it can easily be removed.

Reply:
We can keep it 👍